.github/workflows/sonarcloud.yml

```
@@ -31,11 +31,44 @@ jobs:
+          INFOBIP_BASE_URL: ${{ secrets.INFOBIP_BASE_URL }}
+          INFOBIP_SENDER: ${{ secrets.INFOBIP_SENDER }}
+        run: echo "Environment variables set."
+
+      - name: Run Tests and Coverage
         run: |
-          echo "Loading .env file"
-          python -c "import dotenv; dotenv.load_dotenv()"
+          source $GITHUB_ENV
+          python -c "from dotenv import load_dotenv; load_dotenv()"
           pytest --cov=app --cov-report=xml --cov-report=term
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)